apps/src/experiments.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This module contains logic for tracking various experiments. Experiments
+ * can be enabled/disabled one of two ways
+ * (1) enable: localStorage.set('experiments-experimentName', true)
+ *     disable: localStorage.removeItem('experiments-experimentName')
+ * (2) Add a query param, i.e. http://foo.com?experimentName=true or
+ *     http://foo.com?experimentName=false
+ * The latter approach ends up toggling the localStorage state.
+ * As long as you check isEnabled once for some experiment, data for all
+ * experiments will be loaded from localStorage and the query params will be
+ * parsed.
+ * To support a new experiment, just add it as a key in experimentList
+ */
+var experiments = module.exports;
+
+// List of experiements, and their current enabled/disabled state
+var experimentList = {
+  topInstructions: false,
+  runModeIndicators: false
+};
+
+var performedLoad = false;
+
+/**
+ * Checks whether provided experiment is enabled or not
+ * @param {string} key - Name of experiment in question
+ * @returns {bool}
+ */
+experiments.isEnabled = function (key) {
+  if (!performedLoad) {
+    loadFromLocalStorage();
+    processQueryParams(window.location.search);
+    performedLoad = true;
+  }
+  return experimentList[key] === true;
+};
+
+/**
+ * Looks at query params and sees if we need to toggle the state of any of our
+ * experiments
+ */
+function processQueryParams(queryString) {
+  Object.keys(experimentList).forEach(function (key) {
+    var regex = new RegExp(key + '=(true|false)');
```

Comment:
Though not strictly necessary, it might be nice to make experiment names (and true/false values) case-insensitive.  Also to accept 0 or 1 as values.